### PR TITLE
Add looking for a team option to projects form 

### PR DESCRIPTION
**Pull request by zoek1**
##### Description
- [X] Add option and input to indicate the user is looking for team members
- [X] Add one image field add one banner to hackathons
- [X] remove discord from project form

##### Refers/Fixes
No issue

##### Testing
Sia Hackathon Projects _ Gitcoin

image

### CODE REVIEW

File: app/dashboard/views.py [hunk 1]
```diff
@@ -3939,24 +3943,24 @@ def hackathon_save_project(request):
                 'logo': request.FILES.get('logo', project.first().logo)
             })
 
-            project.update(**kwargs)
+            # project.update(**kwargs)
```
Comment (octavioamu) [inline app/dashboard/views.py:3946]: the code doesn't need to update the project?

File: app/dashboard/views.py [hunk 2]
```diff
@@ -3939,24 +3943,24 @@ def hackathon_save_project(request):
...
 
-            try:
-                bounty_profile = Profile.objects.get(handle=project.bounty.bounty_owner_github_username.lower())
-                if bounty_profile.chat_id is '' or bounty_profile.chat_id is None:
-                    created, bounty_profile = associate_chat_to_profile(bounty_profile)
+            #try:
```
Comment (octavioamu) [inline app/dashboard/views.py:3948]: we are removing the chat associated actions?

Comment (zoek1) [reply app/dashboard/views.py]: Thanks, i thought i restored these lines. The chat related actions caused crash my environment so i commented this piece of code.

File: app/dashboard/models.py [hunk 3]
```diff
@@ -4668,6 +4669,12 @@ class HackathonProject(SuperModel):
         choices=PROJECT_STATUS,
         blank=True
     )
+    message = models.CharField(
+        max_length=150,
```
Comment (danlipert) [inline app/dashboard/models.py:4673]: I'm wondering if this limit is too short, what do you think?

Comment (zoek1) [reply app/dashboard/models.py:4673]: i think is ok for a short message, originally the tweets have a length of 140 characters and worked well. One example could be:

`Hi! I'm python developer and I'm looking for a fronted partner to hack on the ETC bounty "Create A DIY Hardware Wallet Using Signatory"`

150 character limit will help to synthesize messages, and having a character count helper will help a lot the user. What do you think @PixelantDesign?

**Comment by PixelantDesign** [comment]
check it out ^ @alexvotofuture hopefully this helps with what we talked about last week!

File: app/dashboard/views.py (hunk 2 above)
Comment (androolloyd) [reply app/dashboard/views.py]: @zoek1 there are some docs for the Gitcoin Chat setup, the container will spin up automatically.

If you go through that setup then you'll stop getting any connection issues.

Comment (zoek1) [reply app/dashboard/views.py]: Thanks @androolloyd, I'll check it later today!

**Pull request merged**